docs/source/exporting.rst

```diff
@@ -48,8 +48,6 @@ http://nbconvert.readthedocs.io/en/latest/api/exporters.html
 
 .. autoclass:: LenvsHTMLExporter
```

Comment:
I think that `LenvsHTMLExporter` can be removed too (exporter from latex_envs to HTML)